rskj-core/src/test/java/co/rsk/peg/bitcoin/ErpRedeemScriptTestUtils.java

```diff
@@ -0,0 +1,60 @@
+package co.rsk.peg.bitcoin;
+
+import co.rsk.bitcoinj.core.BtcECKey;
+import co.rsk.bitcoinj.core.Utils;
+import co.rsk.bitcoinj.script.ScriptOpCodes;
+
+import java.math.BigInteger;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ErpRedeemScriptTestUtils {
+    public static void assertStandardMultiSigRedeemScript(byte[] redeemScript, List<BtcECKey> emergencyPublicKeys, int opNumMErpIndex) {
+        int erpThreshold = emergencyPublicKeys.size() / 2 + 1;
```

Comment:
```suggestion
        int threshold = publicKeys.size() / 2 + 1;
```
i think there are some references to erp that shouldnt be here :)

Reply:
done!

Reply:
indeed, this method is for a standard multisig redeem script